cou/cli.py

```diff
@@ -34,7 +34,7 @@
 AVAILABLE_OPTIONS = "cas"
 
 # NOTE(gabrielcocenza) top-level module to reference the same logger object across modules.
-logger = logging.getLogger("cou")
+logger = logging.getLogger(__package__)
```

Comment:
There is no reason to not use `__name__` here, because it will make much more sense to see that logs in this file are coming from this file.

Reply:
True. I've fixed it.